terraform/main.tf

```diff
@@ -0,0 +1,139 @@
+terraform {
+  backend "gcs" {
+    bucket = "f3441e415e6e5e7d-bucket-tfstate"
+    prefix = "terraform/state"
```

Comment:
If we keep this prefix, I think terraform will be confused by all the state that is already in `terraform/state/default.tfstate`. If you run `tf plan`, it will probably want to delete all the stuff we've defined for `pudl` and replace it with all the moz stuff.

We could potentially make `terraform/state/mozilla/default.tfstate` but I think that starts to get confusing - "which piece of infrastructure is governed by which piece of terraform configuration" becomes hard to pin down.